recipe/meta.yaml

```diff
@@ -29,6 +29,9 @@ requirements:
     - python
     - setuptools-rust
   run:
+    # Recipe failed when compiling pyo3 with complaint about linked lib built for SDK 10.12
+    # https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.12") }}  # [osx and x86_64]
```

Comment:
You don't actually need the runtime requirement, because you're only bumping `MACOSX_SDK_VERSION`, not `MACOSX_DEPLOYMENT_TARGET`. Basically, you need a newer SDK to compile the package successfully, but you need nothing newer than 10.9 to _run_ the thing.

Reply:
Neat. Thx